src/site/layouts/office.drupal.liquid

```diff
@@ -24,71 +24,89 @@
           <div class="vads-l-grid-container--full--full">
             <h1>{{ title }}</h1>
             <div class="vads-l-grid-container--full--full">
-              {{ fieldDescription }}
-              {% if title == 'Events' %}
+              <div class="va-introtext">
+                <p class="office-show" id="office-main-description">
+                  {{ fieldDescription }}
+                </p>
+                {% if fieldAssetLibraryDescription %}
```

Comment:
For my own edification: This comes from the `graphql` query, right?

Reply:
Yep, @cvalarida